tests/components/bond/test_init.py

```diff
@@ -27,29 +21,25 @@ async def test_async_setup_no_domain_config(hass: HomeAssistant):
 async def test_async_setup_entry_sets_up_hub_and_supported_domains(hass: HomeAssistant):
     """Test that configuring entry sets up cover domain."""
     config_entry = MockConfigEntry(
-        domain=DOMAIN, data={CONF_HOST: "1.1.1.1", CONF_ACCESS_TOKEN: "test-token"},
+        domain=DOMAIN, data={CONF_HOST: "some host", CONF_ACCESS_TOKEN: "test-token"},
     )
 
-    with patch_bond_device_ids(), patch_setup_entry(
-        "cover"
-    ) as mock_cover_async_setup_entry, patch_setup_entry(
-        "fan"
-    ) as mock_fan_async_setup_entry, patch_setup_entry(
-        "light"
-    ) as mock_light_async_setup_entry, patch_setup_entry(
-        "switch"
-    ) as mock_switch_async_setup_entry:
-        result = await setup_bond_entity(
-            hass,
-            config_entry,
-            hub_version={
-                "bondid": "test-bond-id",
-                "target": "test-model",
-                "fw_ver": "test-version",
-            },
-        )
-        assert result is True
-        await hass.async_block_till_done()
+    with patch_bond_version(
+        return_value={
+            "bondid": "test-bond-id",
+            "target": "test-model",
+            "fw_ver": "test-version",
+        }
+    ):
+        with patch_setup_entry("cover") as mock_cover_async_setup_entry:
```

Comment:
cant we do multiple patch per with vs nested?

Reply:
changed all.